src/core/qgsquick/qgsquickmapsettings.cpp

```diff
@@ -269,8 +269,9 @@ QSize QgsQuickMapSettings::outputSize() const
 
 void QgsQuickMapSettings::setOutputSize( QSize outputSize )
 {
-  outputSize.setWidth( outputSize.width() * devicePixelRatio() );
-  outputSize.setHeight( outputSize.height() * devicePixelRatio() );
+  outputSize.setWidth( std::ceil( outputSize.width() * devicePixelRatio() ) );
+  outputSize.setHeight( std::ceil( outputSize.height() * devicePixelRatio() ) );
+  qDebug() << outputSize;
```

Comment:
QDebug.

Reply:
Ouf thanks god for Windows' long CI build time, I had it on auto merge ;)